src/main.js

```diff
@@ -177,70 +177,255 @@ let destinations = [
     {
         city: 'Paris',
         country: 'France',
-        photo: 'img/paris.jpg',
+        photo: require('./img/paris.jpg'),
         visited: false
     },
     {
         city: 'Santorini',
         country: 'Greece',
-        photo: 'img/santorini.jpg',
+        photo: require('./img/santorini.jpg'),
         visited: false
     },
     {
         city: 'New York',
         country: 'U.S.',
-        photo: 'img/newyork.jpg',
+        photo: require('./img/newyork.jpg'),
         visited: false
     },
     {
         city: 'Marrakech',
         country: 'Morocco',
-        photo: 'img/marrakech.jpg',
+        photo: require('./img/marrakech.jpg'),
         visited: false
     },
     {
         city: 'Amsterdam',
         country: 'Netherlands',
-        photo: 'img/Amsterdam.png',
+        photo: require('./img/Amsterdam.png'),
         visited: false
     },
     {
         city: 'Berlin',
         country: 'Germany',
-        photo: 'img/berlin.jpg',
+        photo: require('./img/berlin.jpg'),
         visited: false
     },
     {
         city: 'Lisbon',
         country: 'Porugal',
-        photo: 'img/lisbon.jpg',
+        photo: require('./img/lisbon.jpg'),
         visited: false
     },
     {
         city: 'Tokyo',
         country: 'Japan',
-        photo: 'img/tokyo.jpg',
+        photo: require('./img/tokyo.jpg'),
         visited: false
     },
     {
         city: 'Sienna',
         country: 'Italy',
-        photo: 'img/nice.jpg',
+        photo: require('./img/sienna.jpg'),
         visited: false
 
     },
     {
         city: 'Dublin',
         country: 'Ireland',
-        photo: 'img/dublin.jpg',
+        photo: require('./img/dublin.jpg'),
         visited: false
     },
     {
         city: 'Saint petersburg',
         country: 'Russia',
-        photo: 'img/saintpetersburg.jpg',
+        photo: require('./img/saintpetersburg.jpg'),
         visited: false
     },
   ]
 
 
+  //visited and bucktelist lists
+  // const visited = destinations.filter(item => item.visited === true);
+  // const bucketlist = destinations.filter(item => item.visited === false);
+ 
+
+  //get list of urls
+  const getURL = (destinationsList) => {
+    let url= destinationsList.map(obj => obj.photo.valueOf());
+    return url;
+  }
+
+  //create elements for each city
+  const elements= destinationsList.forEach(destination => {
```

Comment:
This `elements` has nothing. A `forEach` does not return anything.

Reply:
Not sure what to do about it since deleting it breaks the gallery?

Reply:
You already did it :D Now there's just the `destinationsList.forEach()` without assigning it to a const, and that's the right way of doing it ;)

Reply:
Oh I see 😄 I didn't get you correctly